src/lib/artworkToSubmissionMapping.ts

```diff
@@ -0,0 +1,70 @@
+const ValidSubmissionCategories = [
+  "ARCHITECTURE",
+  "DESIGN_DECORATIVE_ART",
+  "DRAWING_COLLAGE_OR_OTHER_WORK_ON_PAPER",
+  "FASHION_DESIGN_AND_WEARABLE_ART",
+  "INSTALLATION",
+  "JEWELRY",
+  "MIXED_MEDIA",
+  "PAINTING",
+  "PERFORMANCE_ART",
+  "PHOTOGRAPHY",
+  "PRINT",
+  "SCULPTURE",
+  "TEXTILE_ARTS",
+  "VIDEO_FILM_ANIMATION",
+]
```

Comment:
This makes me cringe a bit knowing that we already have [many versions of this mapping across our repos](https://github.com/search?q=org%3Aartsy+FASHION_DESIGN_AND_WEARABLE_ART&type=code) 😬 

I hope some day soon we won't need a different set of accepted categories between artworks and consignment submissions. Category (Medium Type) is one of those attributes that I'm hoping we can defer fully to the Artwork model and deprecate its presence in the Submission model entirely. 😅

Reply:
I was under the impression only these are the valid categories we accept
```typescript
{
  OTHER: "OTHER",
  PAINTING: "PAINTING",
  DRAWING_COLLAGE_OR_OTHER_WORK_ON_PAPER: "DRAWING_COLLAGE_OR_OTHER_WORK_ON_PAPER",
  PHOTOGRAPHY: "PHOTOGRAPHY",
  PRINT: "PRINT",
  SCULPTURE: "SCULPTURE",
}
```

Reply:
Yeah – and that does align with available artwork mediums defined [here](https://github.com/artsy/metaphysics/blob/9ccad9c469d57897140a20e8bec62af4ff3d3ff9/src/lib/artworkMediums.ts#L4) in Metaphysics.

I think there's some friction between this MP-backed list, which I'd prefer to use in the UI, and the enum defined in Convection. I'm thinking soon we'll be able to defer to the MyC artwork for this medium type and we can simply stop using the [Convection-backed enum](https://github.com/artsy/convection/blob/45f7d5984bc26ad90a9ed8ef92bb28f60facccfd/app/graphql/types/category_type.rb#L4).